src/components/ImageCard.react.js

```diff
@@ -147,8 +159,52 @@ var ImageCard = React.createClass({
         <span className="icon icon-badge-private"></span>
       );
     }
-    return (
-      <div className="image-item">
+    let create;
+    let overlay;
+    if (this.props.image.is_local) {
```

Comment:
Makes sense

I think the best workaround is to eventually use subcomponents, but that's for another PR.